frontend/src/components/worksheets/WorksheetItemList.js

```diff
@@ -204,11 +204,16 @@ class WorksheetItemList extends React.Component {
         // Create items
         var items_display;
         var info = this.props.ws.info;
+        if (info && info.items.length === 0) { // Create a "dummy" item at the beginning so that only empty text can be added.
+            info.items = [{"isDummyItem": true, "text":"","mode":"markup_block","sort_keys":[-1],"ids":[null],"is_refined":true}];
```

Comment:
Make spacing consistent, also might be easier to read with multiple lines.

Reply:
Updated, although for these things, I was thinking of adding `prettier` to our `lint-staged` step (https://github.com/codalab/codalab-worksheets/blob/5afec265baa00c3d4e8bd76e372472364f48c1ea/frontend/package.json#L70). This would need to wait until the frontend is merged though so that the diff between master and frontend isn't too great.